lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -402,6 +408,19 @@ newtype ApiSelectCoinsData (n :: NetworkDiscriminant) = ApiSelectCoinsData
     { payments :: NonEmpty (AddressAmount (ApiT Address, Proxy n))
     } deriving (Eq, Generic, Show)
 
+data ApiSelectCoinsActionData (n :: NetworkDiscriminant)
+    = ApiSelectCoinsActionData
+    { action :: W.DelegationAction
+    , passphrase :: ApiT (Passphrase "raw")
```

Comment:
As outlined in other comments, the `passphrase` should not be required.

Reply:
I am totally out of my depth on this one.